action.sh

```diff
@@ -243,8 +299,8 @@ systemctl enable shutdown.service
   if [[ $GH_READY == 1 ]]; then
     echo "✅ ${VM_ID} ready ..."
   else
-    echo "Waited 2 minutes for ${VM_ID}, without luck, deleting ${VM_ID} ..."
-    gcloud --quiet compute instances delete ${VM_ID} --zone=${machine_zone}
+    echo "Waited 5 minutes for ${VM_ID}, without luck, deleting ${VM_ID} ..."
+    # gcloud --quiet compute instances delete ${VM_ID} --zone=${machine_zone}
```

Comment:
Why commented out?

Reply:
To aid debugging.  Fixed with [945578f](https://github.com/related-sciences/gce-github-runner/pull/33/commits/945578fb8561353f87bb9984fdee7cc73f803a78).